src/layout/Input/InputComponent.tsx

```diff
@@ -82,9 +87,13 @@ export const InputComponent: React.FunctionComponent<IInputProps> = ({ node, isV
 
   const { langAsString } = useLanguage();
 
-  const reactNumberFormatConfig = useMapToReactNumberConfig(formatting as IInputFormatting | undefined, formValue);
+  const reactNumberFormatConfig = useMapToReactNumberConfig(
+    formatting as IInputFormattingInternal | undefined,
+    formValue,
+  );
   const ariaLabel = overrideDisplay?.renderedInTable === true ? langAsString(textResourceBindings?.title) : undefined;
-
+  const prefixExternal = textResourceBindings?.prefix ? langAsString(textResourceBindings.prefix) : undefined;
+  const suffixExternal = textResourceBindings?.suffix ? langAsString(textResourceBindings.suffix) : undefined;
```

Comment:
Ah, I was a bit confused at first - so these are not 'external' as in internal/external types (aka after/before expression evaluation), but visually external to the Input field? How about `prefixText` or 'altPrefix`? :thinking: Or, i dunno, external also works when I know about how it's meant.

Reply:
Yeah sure, just needed a name that did not conflict 😅